### PR TITLE
Changing the Extract Logic of continuation_token

### PR DESCRIPTION
**Pull request by rajbhupendra588**
Changing the Extract Logic of `continuation_token`

### CODE REVIEW

File: src/main/java/org/springframework/social/partnercenter/api/PartnerCenterResponse.java [hunk 1]
```diff
@@ -58,12 +61,23 @@ public PartnerCenterResponse setLinks(ResourceLinks links) {
 	}
 
 	public String getContinuationToken() {
-		return continuationToken;
+		return extractContinuationToken().orElse(null);
```
Comment (jpboudreault) [inline src/main/java/org/springframework/social/partnercenter/api/PartnerCenterResponse.java]: Does this work for the customer list? AFAIK the customer list response still has the old parameter.

File: src/main/java/org/springframework/social/partnercenter/api/PartnerCenterResponse.java [hunk 2]
```diff
@@ -58,12 +61,23 @@ public PartnerCenterResponse setLinks(ResourceLinks links) {
...
 	}
 
-	public PartnerCenterResponse setContinuationToken(String continuationToken) {
```
Comment (jpboudreault) [inline src/main/java/org/springframework/social/partnercenter/api/PartnerCenterResponse.java]: Was this used somewhere?

File: src/main/java/org/springframework/social/partnercenter/api/PartnerCenterResponse.java (hunk 1 above)
Comment (jpboudreault) [inline src/main/java/org/springframework/social/partnercenter/api/PartnerCenterResponse.java]: The old field was used for the fetch customer list. We would need to confirm the changes that you are proposing would not break the sync functionality

File: src/main/java/org/springframework/social/partnercenter/api/PartnerCenterResponse.java (hunk 2 above)
Comment (rajbhupendra588) [reply src/main/java/org/springframework/social/partnercenter/api/PartnerCenterResponse.java]: > Was this used somewhere?
setContinuationToken() no usages in monolith till now.

**Comment by rajbhupendra588** [comment]
@jonfk  @jpboudreault @paulsmelser , can somebody add me in this REPO , may be this is the reason `continuous-integration/travis-ci` is still running.

**Comment by rajbhupendra588** [comment]
closing PR

**Pull request closed without merging**